mailbag/controller.py

```diff
@@ -214,6 +220,19 @@ def generate_mailbag(self):
                 writer = csv.writer(f)
                 writer.writerows(error_csv)
 
+        #moving companion files
+        if self.args.companion_files:
+            new_path = os.path.join(mailbag_dir, "data", self.args.input)
+            new_files=[]
+            for root, dirs, files in os.walk(parent_dir):
+                for file in files:
+                    new_files.append(file)
```

Comment:
Instead of appending to a list here, you should be able to just move the file from here. Remember, there might be directories such as:
```
parent_dir
    \Inbox
        message.eml
        companionMetadata.xlsx
```

So, you have to join `root` and `file` to get the full path. You might be able to use `helper.relativePath()` to get the path relative to `parent_dir`. That's what you would then `os.path.join()` with `new_path`. Then you can use `helper.moveFile()` instead of `shutil.move()` which adds some logging and error handling.